### ISSUE TITLE
Vaadin `implementation` fails the build

### ISSUE THREAD

**Issue by mvysny** [vaadin/vaadin-gradle-plugin#24]
When the vaadin-flow* jars are pulled in as `implementation` dependency, the Vaadin Gradle Plugin will incorrectly compute classpath for Flow+Webpack and the webpack packaging will fail.

To reproduce the bug:
1. git clone https://github.com/vaadin/base-starter-spring-gradle
2. delete `node_modules` if it's present
3. edit `build.gradle` and change the `compile(` to `implementation(`
4. run `./gradlew vaadinBuildFrontend`

**Comment by mvysny** [comment]
The full log:
```
* What went wrong:
Execution failed for task ':vaadinBuildFrontend'.
> Command '/usr/bin/node /home/mavi/work/vaadin/base-starter-spring-gradle/node_modules/webpack/bin/webpack.js' failed with exit code 2: 
  The `corejs` option only has an effect when the `useBuiltIns` option is not `false`
  
  
  The `corejs` option only has an effect when the `useBuiltIns` option is not `false`
  
  Hash: 1bb6d9ce930076fa0671
  Version: webpack 4.30.0
  Time: 15887ms
  Built at: 02/03/2020 1:36:05 PM
   6 assets
  Entrypoint bundle = build/vaadin-bundle-014e61e5de212630bdd0.cache.js
  Entrypoint bundle.es5 = build/vaadin-bundle.es5-4f275e0443332261c56f.cache.js
   [58] ../node_modules/@polymer/polymer/lib/elements/custom-style.js?babel-target=es5 + 1 modules 7.5 KiB {1} [built]
        |    2 modules
   [59] ../node_modules/@polymer/polymer/lib/elements/custom-style.js?babel-target=es6 + 1 modules 6.32 KiB {0} [built]
        |    2 modules
   [81] ./styles/shared-styles.css 327 bytes {0} {1} [built]
   [82] ./styles/vaadin-text-field-styles.css 399 bytes {0} {1} [built]
  [110] ../node_modules/@vaadin/vaadin-text-field/theme/lumo/vaadin-text-field.js?babel-target=es6 + 1 modules 8.64 KiB {0} [built]
        |    2 modules
  [111] ../node_modules/@vaadin/vaadin-text-field/theme/lumo/vaadin-text-field.js?babel-target=es5 + 1 modules 17.7 KiB {1} [built]
        |    2 modules
  [128] ../node_modules/@polymer/iron-icon/iron-icon.js?babel-target=es6 + 1 modules 16 KiB {0} [built]
        |    2 modules
  [129] ../node_modules/@polymer/iron-icon/iron-icon.js?babel-target=es5 + 1 modules 17 KiB {1} [built]
        |    2 modules
  [144] ../target/frontend/generated-flow-imports.js?babel-target=es5 + 8 modules 54.5 KiB {1} [built]
        | ../target/frontend/generated-flow-imports.js?babel-target=es5 2.13 KiB [built]
        |     + 8 hidden modules
  [145] ../target/frontend/generated-flow-imports.js?babel-target=es6 + 8 modules 42.4 KiB {0} [built]
        | ../target/frontend/generated-flow-imports.js?babel-target=es6 2.1 KiB [built]
        |     + 8 hidden modules
  [156] ../node_modules/@vaadin/vaadin-item/theme/lumo/vaadin-item.js?babel-target=es5 + 1 modules 2.55 KiB {2} [built]
        |    2 modules
  [157] ../node_modules/@vaadin/vaadin-item/theme/lumo/vaadin-item.js?babel-target=es6 + 1 modules 2.22 KiB {3} [built]
        |    2 modules
  [168] ../node_modules/@vaadin/vaadin-text-field/theme/lumo/vaadin-password-field.js?babel-target=es6 + 1 modules 1.05 KiB {3} [built]
        |    2 modules
  [304] ../target/frontend/generated-flow-imports-fallback.js 3.56 KiB {5} [built]
  [305] ../target/frontend/generated-flow-imports-fallback.js 3.56 KiB {4} [built]
      + 291 hidden modules
  
  ERROR in ../target/frontend/generated-flow-imports-fallback.js
  Module not found: Error: Can't resolve '@vaadin/flow-frontend/comboBoxConnector-es6.js?babel-target=es5' in '/home/mavi/work/vaadin/base-starter-spring-gradle/target/frontend'
   @ ../target/frontend/generated-flow-imports-fallback.js 3:0-56
   @ ../target/frontend/generated-flow-imports.js?babel-target=es5
  
  ERROR in ../target/frontend/generated-flow-imports-fallback.js
  Module not found: Error: Can't resolve '@vaadin/flow-frontend/comboBoxConnector-es6.js?babel-target=es6' in '/home/mavi/work/vaadin/base-starter-spring-gradle/target/frontend'
   @ ../target/frontend/generated-flow-imports-fallback.js 3:0-56
   @ ../target/frontend/generated-flow-imports.js?babel-target=es6
  
  ERROR in ../target/frontend/generated-flow-imports-fallback.js
  Module not found: Error: Can't resolve '@vaadin/flow-frontend/comboBoxConnector.js?babel-target=es5' in '/home/mavi/work/vaadin/base-starter-spring-gradle/target/frontend'
   @ ../target/frontend/generated-flow-imports-fallback.js 54:0-52
   @ ../target/frontend/generated-flow-imports.js?babel-target=es5
  
  ERROR in ../target/frontend/generated-flow-imports-fallback.js
  Module not found: Error: Can't resolve '@vaadin/flow-frontend/comboBoxConnector.js?babel-target=es6' in '/home/mavi/work/vaadin/base-starter-spring-gradle/target/frontend'
   @ ../target/frontend/generated-flow-imports-fallback.js 54:0-52
   @ ../target/frontend/generated-flow-imports.js?babel-target=es6
  
  ERROR in ../target/frontend/generated-flow-imports-fallback.js
  Module not found: Error: Can't resolve '@vaadin/flow-frontend/contextMenuConnector-es6.js?babel-target=es5' in '/home/mavi/work/vaadin/base-starter-spring-gradle/target/frontend'
   @ ../target/frontend/generated-flow-imports-fallback.js 4:0-59
   @ ../target/frontend/generated-flow-imports.js?babel-target=es5
  
  ERROR in ../target/frontend/generated-flow-imports-fallback.js
  Module not found: Error: Can't resolve '@vaadin/flow-frontend/contextMenuConnector-es6.js?babel-target=es6' in '/home/mavi/work/vaadin/base-starter-spring-gradle/target/frontend'
   @ ../target/frontend/generated-flow-imports-fallback.js 4:0-59
   @ ../target/frontend/generated-flow-imports.js?babel-target=es6
  
  ERROR in ../target/frontend/generated-flow-imports-fallback.js
  Module not found: Error: Can't resolve '@vaadin/flow-frontend/contextMenuConnector.js?babel-target=es5' in '/home/mavi/work/vaadin/base-starter-spring-gradle/target/frontend'
   @ ../target/frontend/generated-flow-imports-fallback.js 55:0-55
   @ ../target/frontend/generated-flow-imports.js?babel-target=es5
  
  ERROR in ../target/frontend/generated-flow-imports-fallback.js
  Module not found: Error: Can't resolve '@vaadin/flow-frontend/contextMenuConnector.js?babel-target=es6' in '/home/mavi/work/vaadin/base-starter-spring-gradle/target/frontend'
   @ ../target/frontend/generated-flow-imports-fallback.js 55:0-55
   @ ../target/frontend/generated-flow-imports.js?babel-target=es6
  
  ERROR in ../target/frontend/generated-flow-imports-fallback.js
  Module not found: Error: Can't resolve '@vaadin/flow-frontend/datepickerConnector.js?babel-target=es5' in '/home/mavi/work/vaadin/base-starter-spring-gradle/target/frontend'
   @ ../target/frontend/generated-flow-imports-fallback.js 5:0-54
   @ ../target/frontend/generated-flow-imports.js?babel-target=es5
  
  ERROR in ../target/frontend/generated-flow-imports-fallback.js
  Module not found: Error: Can't resolve '@vaadin/flow-frontend/datepickerConnector.js?babel-target=es6' in '/home/mavi/work/vaadin/base-starter-spring-gradle/target/frontend'
   @ ../target/frontend/generated-flow-imports-fallback.js 5:0-54
   @ ../target/frontend/generated-flow-imports.js?babel-target=es6
  
  ERROR in ../target/frontend/generated-flow-imports-fallback.js
  Module not found: Error: Can't resolve '@vaadin/flow-frontend/dndConnector-es6.js?babel-target=es5' in '/home/mavi/work/vaadin/base-starter-spring-gradle/target/frontend'
   @ ../target/frontend/generated-flow-imports-fallback.js 6:0-51
   @ ../target/frontend/generated-flow-imports.js?babel-target=es5
  
  ERROR in ../target/frontend/generated-flow-imports-fallback.js
  Module not found: Error: Can't resolve '@vaadin/flow-frontend/dndConnector-es6.js?babel-target=es6' in '/home/mavi/work/vaadin/base-starter-spring-gradle/target/frontend'
   @ ../target/frontend/generated-flow-imports-fallback.js 6:0-51
   @ ../target/frontend/generated-flow-imports.js?babel-target=es6
  
  ERROR in ../target/frontend/generated-flow-imports-fallback.js
  Module not found: Error: Can't resolve '@vaadin/flow-frontend/dndConnector.js?babel-target=es5' in '/home/mavi/work/vaadin/base-starter-spring-gradle/target/frontend'
   @ ../target/frontend/generated-flow-imports-fallback.js 56:0-47
   @ ../target/frontend/generated-flow-imports.js?babel-target=es5
  
  ERROR in ../target/frontend/generated-flow-imports-fallback.js
  Module not found: Error: Can't resolve '@vaadin/flow-frontend/dndConnector.js?babel-target=es6' in '/home/mavi/work/vaadin/base-starter-spring-gradle/target/frontend'
   @ ../target/frontend/generated-flow-imports-fallback.js 56:0-47
   @ ../target/frontend/generated-flow-imports.js?babel-target=es6
  
  ERROR in ../target/frontend/generated-flow-imports.js?babel-target=es5
  Module not found: Error: Can't resolve '@vaadin/flow-frontend/flow-component-renderer.js?babel-target=es5' in '/home/mavi/work/vaadin/base-starter-spring-gradle/target/frontend'
   @ ../target/frontend/generated-flow-imports.js?babel-target=es5 15:0-58
  
  ERROR in ../target/frontend/generated-flow-imports.js?babel-target=es6
  Module not found: Error: Can't resolve '@vaadin/flow-frontend/flow-component-renderer.js?babel-target=es6' in '/home/mavi/work/vaadin/base-starter-spring-gradle/target/frontend'
   @ ../target/frontend/generated-flow-imports.js?babel-target=es6 15:0-58
  
  ERROR in ../target/frontend/generated-flow-imports-fallback.js
  Module not found: Error: Can't resolve '@vaadin/flow-frontend/gridConnector-es6.js?babel-target=es5' in '/home/mavi/work/vaadin/base-starter-spring-gradle/target/frontend'
   @ ../target/frontend/generated-flow-imports-fallback.js 7:0-52
   @ ../target/frontend/generated-flow-imports.js?babel-target=es5
  
  ERROR in ../target/frontend/generated-flow-imports-fallback.js
  Module not found: Error: Can't resolve '@vaadin/flow-frontend/gridConnector-es6.js?babel-target=es6' in '/home/mavi/work/vaadin/base-starter-spring-gradle/target/frontend'
   @ ../target/frontend/generated-flow-imports-fallback.js 7:0-52
   @ ../target/frontend/generated-flow-imports.js?babel-target=es6
  
  ERROR in ../target/frontend/generated-flow-imports-fallback.js
  Module not found: Error: Can't resolve '@vaadin/flow-frontend/gridConnector.js?babel-target=es5' in '/home/mavi/work/vaadin/base-starter-spring-gradle/target/frontend'
   @ ../target/frontend/generated-flow-imports-fallback.js 57:0-48
   @ ../target/frontend/generated-flow-imports.js?babel-target=es5
  
  ERROR in ../target/frontend/generated-flow-imports-fallback.js
  Module not found: Error: Can't resolve '@vaadin/flow-frontend/gridConnector.js?babel-target=es6' in '/home/mavi/work/vaadin/base-starter-spring-gradle/target/frontend'
   @ ../target/frontend/generated-flow-imports-fallback.js 57:0-48
   @ ../target/frontend/generated-flow-imports.js?babel-target=es6
  
  ERROR in ../target/frontend/generated-flow-imports-fallback.js
  Module not found: Error: Can't resolve '@vaadin/flow-frontend/ironListConnector-es6.js?babel-target=es5' in '/home/mavi/work/vaadin/base-starter-spring-gradle/target/frontend'
   @ ../target/frontend/generated-flow-imports-fallback.js 8:0-56
   @ ../target/frontend/generated-flow-imports.js?babel-target=es5
  
  ERROR in ../target/frontend/generated-flow-imports-fallback.js
  Module not found: Error: Can't resolve '@vaadin/flow-frontend/ironListConnector-es6.js?babel-target=es6' in '/home/mavi/work/vaadin/base-starter-spring-gradle/target/frontend'
   @ ../target/frontend/generated-flow-imports-fallback.js 8:0-56
   @ ../target/frontend/generated-flow-imports.js?babel-target=es6
  
  ERROR in ../target/frontend/generated-flow-imports-fallback.js
  Module not found: Error: Can't resolve '@vaadin/flow-frontend/ironListConnector.js?babel-target=es5' in '/home/mavi/work/vaadin/base-starter-spring-gradle/target/frontend'
   @ ../target/frontend/generated-flow-imports-fallback.js 58:0-52
   @ ../target/frontend/generated-flow-imports.js?babel-target=es5
  
  ERROR in ../target/frontend/generated-flow-imports-fallback.js
  Module not found: Error: Can't resolve '@vaadin/flow-frontend/ironListConnector.js?babel-target=es6' in '/home/mavi/work/vaadin/base-starter-spring-gradle/target/frontend'
   @ ../target/frontend/generated-flow-imports-fallback.js 58:0-52
   @ ../target/frontend/generated-flow-imports.js?babel-target=es6
  
  ERROR in ../target/frontend/generated-flow-imports-fallback.js
  Module not found: Error: Can't resolve '@vaadin/flow-frontend/ironListStyles.js?babel-target=es5' in '/home/mavi/work/vaadin/base-starter-spring-gradle/target/frontend'
   @ ../target/frontend/generated-flow-imports-fallback.js 9:0-49
   @ ../target/frontend/generated-flow-imports.js?babel-target=es5
  
  ERROR in ../target/frontend/generated-flow-imports-fallback.js
  Module not found: Error: Can't resolve '@vaadin/flow-frontend/ironListStyles.js?babel-target=es6' in '/home/mavi/work/vaadin/base-starter-spring-gradle/target/frontend'
   @ ../target/frontend/generated-flow-imports-fallback.js 9:0-49
   @ ../target/frontend/generated-flow-imports.js?babel-target=es6
  
  ERROR in ../target/frontend/generated-flow-imports-fallback.js
  Module not found: Error: Can't resolve '@vaadin/flow-frontend/menubarConnector.js?babel-target=es5' in '/home/mavi/work/vaadin/base-starter-spring-gradle/target/frontend'
   @ ../target/frontend/generated-flow-imports-fallback.js 10:0-51
   @ ../target/frontend/generated-flow-imports.js?babel-target=es5
  
  ERROR in ../target/frontend/generated-flow-imports-fallback.js
  Module not found: Error: Can't resolve '@vaadin/flow-frontend/menubarConnector.js?babel-target=es6' in '/home/mavi/work/vaadin/base-starter-spring-gradle/target/frontend'
   @ ../target/frontend/generated-flow-imports-fallback.js 10:0-51
   @ ../target/frontend/generated-flow-imports.js?babel-target=es6
  
  ERROR in ../target/frontend/generated-flow-imports-fallback.js
  Module not found: Error: Can't resolve '@vaadin/flow-frontend/selectConnector.js?babel-target=es5' in '/home/mavi/work/vaadin/base-starter-spring-gradle/target/frontend'
   @ ../target/frontend/generated-flow-imports-fallback.js 59:0-50
   @ ../target/frontend/generated-flow-imports.js?babel-target=es5
  
  ERROR in ../target/frontend/generated-flow-imports-fallback.js
  Module not found: Error: Can't resolve '@vaadin/flow-frontend/selectConnector.js?babel-target=es6' in '/home/mavi/work/vaadin/base-starter-spring-gradle/target/frontend'
   @ ../target/frontend/generated-flow-imports-fallback.js 59:0-50
   @ ../target/frontend/generated-flow-imports.js?babel-target=es6
  
  ERROR in ../target/frontend/generated-flow-imports-fallback.js
  Module not found: Error: Can't resolve '@vaadin/flow-frontend/timepickerConnector.js?babel-target=es5' in '/home/mavi/work/vaadin/base-starter-spring-gradle/target/frontend'
   @ ../target/frontend/generated-flow-imports-fallback.js 11:0-54
   @ ../target/frontend/generated-flow-imports.js?babel-target=es5
  
  ERROR in ../target/frontend/generated-flow-imports-fallback.js
  Module not found: Error: Can't resolve '@vaadin/flow-frontend/timepickerConnector.js?babel-target=es6' in '/home/mavi/work/vaadin/base-starter-spring-gradle/target/frontend'
   @ ../target/frontend/generated-flow-imports-fallback.js 11:0-54
   @ ../target/frontend/generated-flow-imports.js?babel-target=es6
  
  ERROR in ../target/frontend/generated-flow-imports-fallback.js
  Module not found: Error: Can't resolve '@vaadin/flow-frontend/vaadin-big-decimal-field.js?babel-target=es5' in '/home/mavi/work/vaadin/base-starter-spring-gradle/target/frontend'
   @ ../target/frontend/generated-flow-imports-fallback.js 12:0-59
   @ ../target/frontend/generated-flow-imports.js?babel-target=es5
  
  ERROR in ../target/frontend/generated-flow-imports-fallback.js
  Module not found: Error: Can't resolve '@vaadin/flow-frontend/vaadin-big-decimal-field.js?babel-target=es6' in '/home/mavi/work/vaadin/base-starter-spring-gradle/target/frontend'
   @ ../target/frontend/generated-flow-imports-fallback.js 12:0-59
   @ ../target/frontend/generated-flow-imports.js?babel-target=es6
  
  ERROR in ../target/frontend/generated-flow-imports-fallback.js
  Module not found: Error: Can't resolve '@vaadin/flow-frontend/vaadin-grid-flow-selection-column.js?babel-target=es5' in '/home/mavi/work/vaadin/base-starter-spring-gradle/target/frontend'
   @ ../target/frontend/generated-flow-imports-fallback.js 13:0-68
   @ ../target/frontend/generated-flow-imports.js?babel-target=es5
  
  ERROR in ../target/frontend/generated-flow-imports-fallback.js
  Module not found: Error: Can't resolve '@vaadin/flow-frontend/vaadin-grid-flow-selection-column.js?babel-target=es6' in '/home/mavi/work/vaadin/base-starter-spring-gradle/target/frontend'
   @ ../target/frontend/generated-flow-imports-fallback.js 13:0-68
   @ ../target/frontend/generated-flow-imports.js?babel-target=es6
```

**Comment by mvysny** [comment]
Fixed in Vaadin-Gradle-Plugin 0.4.0
